### ISSUE TITLE
`isort` ordering of imports not being enforced.

### ISSUE THREAD

**Issue by brysontyrrell** [charliermarsh/ruff#3188]
I'm experimenting replacing `isort` with `ruff` but it doesn't seem to be enforcing the order of imports even if I explicitly set `order-by-type = true`.

Here's what is in `pyproject.toml`:

```toml
[tool.ruff]
line-length = 100
target-version = "py39"
src = [
    "src",
    "test"
]


[tool.ruff.isort]
known-first-party = [ "my-package" ]
order-by-type = true
force-sort-within-sections = true
```

Here's the test Python file `my-package/__init__.py`:
```python
from .shared import IMPORT_ME  # first-party

import json  # standard lib
import requests  # third party


print(requests)
print(json)
print(IMPORT_ME)
```

`isort` properly returns an error when it checks this file:
```
ERROR: /path/to/my-package/__init__.py Imports are incorrectly sorted and/or formatted
```

Ruff is at version `0.0.252`.

**Comment by brysontyrrell** [comment]
Ignore that I opened this...

**Comment by charliermarsh** [comment]
All good! Is it just that you needed to enable the `I` rules?

**Comment by brysontyrrell** [comment]
I had it in the wrong place in the pyproject file 🤦‍♂️ Saw it after I hit save on the issue.

**Comment by charliermarsh** [comment]
Happens to the best of us :D
